src/options.ts

```diff
@@ -6,5 +6,4 @@ export interface WebRTCListenerOptions extends CreateListenerOptions {
   //   channelOptions?: WebRTCReceiverInit
 }
 
-export interface WebRTCDialOptions extends DialOptions {
-}
+export interface WebRTCDialOptions extends DialOptions {}
```

Comment:
our linting rules must be pretty relaxed as normally you would get an "empty interface" complaint here, no?